internal/configuration/viper_test.go

```diff
@@ -0,0 +1,378 @@
+package configuration
+
+import (
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/util"
+)
+
+func TestConfig_GetDebugMode(t *testing.T) {
+	os.Setenv(DebugMode, "false")
+	tests := []struct {
+		name string
+		want bool
+	}{
+		{
+			name: "Config Debug",
+			want: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			c, _ := New("")
+			if got := c.GetDebugMode(); got != tt.want {
```

Comment:
minor: idler is already using `testify/assert` which you can make use of to `assert.Equal(t, expected, actual, msg)`